tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/character_preferences/language.tsx

```diff
@@ -2,5 +2,7 @@ import { FeatureChoiced, FeatureDropdownInput } from '../base';
 
 export const language: FeatureChoiced = {
   name: 'Language',
+  description:
+    "The extra language you know. Note that if you already know the language, you'll be given galactic uncommon instead.",
```

Comment:
I don't think the extra addendum here is necessary which means I think this whole thing is unnecessary